openapi/openapi.yaml

```diff
@@ -999,6 +976,176 @@ components:
           required:
             - data
 
+    UiCollection:
+      title: 'UiCollection'
+      description: 'Detailed info about a collection used by the ui'
+      type: object
+      properties:
+        all_versions:
+          readOnly: true
+          type: array
+          items:
+            $ref: '#/components/schemas/UiCollectionVersion'
+        community_score:
+          nullable: true
+          type: number
+        community_survey_count:
+          type: integer
+        deprecated:
+          type: boolean
+        download_count:
+          type: integer
+        id:
```

Comment:
@newswangerd does webui ever use these ids? If not, we could get rid of them.

Reply:
Lets keep the ids. They're useful for finding things in lists of objects.